core/src/sql/statements/define/access.rs

```diff
@@ -144,6 +141,38 @@ impl Display for DefineAccessStatement {
 				write!(f, " WITH JWT {}", ac.jwt)?;
 			}
 		}
+		// Always print relevant durations so defaults can be changed in the future
+		// If default values were not printed, exports would not be forward compatible
+		// None values need to be printed, as they are different from the default values
+		write!(f, " DURATION")?;
+		if self.kind.can_issue_grants() {
+			write!(
+				f,
+				" FOR GRANT {},",
```

Comment:
Ideally, we wouldn't print the comma if we don't need to

Reply:
I intended the `DURATION` statement to follow the same behavior as the `PERMISSIONS` clause, which does print the comma for readability even though it is not required: https://github.com/surrealdb/surrealdb/blob/main/core/src/sql/permission.rs#L96-L102